docs/products/flink/howto/create-flink-applications.rst

```diff
@@ -0,0 +1,58 @@
+
+Create an application via the Aiven console
+--------------------------------------------
+
+Follow these steps to build your first Aiven for Flink application: 
+
+1. In the `Aiven Console <https://console.aiven.io/>`_, open the Aiven for Apache Flink service for which you want to create an application. 
+2. Go to the **Applications** tab and click **Create application** to create your first application. 
+3. In the **Create new application** screen, enter the name of your application and configure the necessary deployment settings. Click **Create application**. 
+4. Click **Create first version** to create the first version of the application. 
+5. Click **Add your first source table** to add a source table. 
+   
+   .. note::
+    Since this is your first application, there are currently no other applications where you can import source tables.   
+
+6. In the **Add new source table** screen, 
+    
+    * Select the **Integrated service** from the drop-down list. 
+    * In the **Table SQL** section, enter the statement that will be used to create the table. 
+    * Optionally, click **Run** to view how data is being pulled from the data source. This could take some time based on the data and the connection. 
+    * Click **Add table**. 
+7. Click **Next** to add the sink table, and then click **Add your first sink table**. 
+   
+   .. note::   
+    Since this is your first application, there are currently no other applications where you can import sink tables.
+    
+8.  In the **Add new sink table** screen, 
+    
+    * Select the **Integrated service** from the drop-down list. 
+    * In the **Table SQL** section, enter the statement that will be used to create the table.  
+    * Optionally, click **Run** to view how data is being pulled from the data source. This could take some time based on the data and the connection. 
+    * Click **Add table**. 
+9.  Click **Next** to enter the SQL statement that transforms the data from the source stream. Optionally, click **Run** to view how data is being pulled from the data source. 
+10. Click **Save and deploy later** to save the application. You will see the application on the landing page that provides you with an overview of your application. 
+
+    .. image:: /images/products/flink/application_landingpage_view.png
+        :scale: 50 %
+        :alt: Application landing page with an view of source table, SQL statement and sink table
+    
+11. Click **Create deployment**. On the **Create new deployment** screen, 
+    
+    * Choose the version you wish to deploy. The default version for the first deployment is **Version: 1**. 
+    * Choose the savepoint from where you want to deploy. No savepoints are available for the first application deployment. 
+    * Use the toggle for **Restart on failure** to enable or disable this option. 
+    * Enter the number of parallel instances you want to have for the task. 
```

Comment:
I believe we need to have a concept about parrallel tasks (or a reference to the Flink docs)

Reply:
Linked it to the  [Parallel task execution](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/execution/parallel/) Apache Flink doc.